Resources/views/line/video.html.twig

```diff
@@ -0,0 +1,27 @@
+{% set videoType = content.getField( 'file' ).value.mimeType|default( 'video/mp4' ) %}
+
```

Comment:
Also, if we add https://github.com/ezsystems/DemoBundle/pull/164/files#diff-7b44bced1c149a4c107e0bb40e7e7e5bR9 to full video, wouldn't we need something similar for line if we plan show videos in line view?